docs/src/main/asciidoc/appcds.adoc

```diff
@@ -0,0 +1,95 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= AppCDS
+include::_attributes.adoc[]
+:categories: core, cloud
+:summary: This reference guide explains how to enable AppCDS with Quarkus.
+
+This reference guide explains how to enable Application Class Data Sharing in your Quarkus applications.
+
+== What is Application Class Data Sharing (AppCDS)?
+
+link:https://docs.oracle.com/en/java/javase/17/vm/class-data-sharing.html[Application Class Data Sharing] is a JVM feature that helps reduce the startup time and memory footprint of a JVM application.
+This is achieved by having the JVM create a pre-processed shared archived of classes that are loaded at startup time. As these classes
+are loaded every time the application starts, AppCDS is a conceptually simple way of improving the application startup time,
+without the application itself having to be coded or configured in a specific way. How much of an improvement depends on many factors, such as the number of classes loaded, the underlying hardware etc.
+
+== Vanilla AppCDS generation
+
+The main downside of using AppCDS is that they are generated by launching the application with special flags and having the JVM generate the archive that can then be used in subsequent application launches.
```

Comment:
Why is that a downside? That looks like a upside to me :)

Reply:
If you like making your life harder, sure, it's an upside :)

Reply:
I mean that this statement looks like an upside, not the process ;)

Reply:
How about now?

Reply:
Looks good. Thanks!